specificatie/gba-genereervariant/openapi.json

```diff
@@ -1024,82 +1024,82 @@
   },
   "components" : {
     "schemas" : {
-      "VerblijfplaatshistorieQueryResponse" : {
+      "HistorieQueryResponseBasis" : {
         "type" : "object",
         "properties" : {
-          "verblijfplaatsen" : {
-            "type" : "array",
-            "items" : {
-              "$ref" : "#/components/schemas/GbaVerblijfplaats"
-            }
+          "opschortingBijhouding" : {
+            "$ref" : "#/components/schemas/GbaOpschortingBijhouding"
           },
           "geheimhoudingPersoonsgegevens" : {
             "$ref" : "#/components/schemas/GbaGeheimhoudingPersoonsgegevens"
           },
           "verificatie" : {
             "$ref" : "#/components/schemas/GbaVerificatie"
           },
-          "opschortingBijhouding" : {
-            "$ref" : "#/components/schemas/GbaOpschortingBijhouding"
+          "rni" : {
+            "$ref" : "#/components/schemas/RniDeelnemer"
           }
         }
       },
+      "VerblijfplaatshistorieQueryResponse" : {
+        "allOf" : [ {
+          "$ref" : "#/components/schemas/HistorieQueryResponseBasis"
+        }, {
+          "type" : "object",
+          "properties" : {
+            "verblijfplaatsen" : {
+              "type" : "array",
+              "items" : {
+                "$ref" : "#/components/schemas/GbaVerblijfplaats"
+              }
+            }
+          }
+        } ]
+      },
       "PartnerhistorieQueryResponse" : {
-        "type" : "object",
-        "properties" : {
-          "partners" : {
-            "type" : "array",
-            "items" : {
-              "$ref" : "#/components/schemas/GbaPartner"
+        "allOf" : [ {
+          "$ref" : "#/components/schemas/HistorieQueryResponseBasis"
+        }, {
+          "type" : "object",
+          "properties" : {
+            "partners" : {
+              "type" : "array",
+              "items" : {
+                "$ref" : "#/components/schemas/GbaPartner"
+              }
             }
-          },
-          "geheimhoudingPersoonsgegevens" : {
-            "$ref" : "#/components/schemas/GbaGeheimhoudingPersoonsgegevens"
-          },
-          "opschortingBijhouding" : {
-            "$ref" : "#/components/schemas/GbaOpschortingBijhouding"
           }
-        }
+        } ]
       },
       "VerblijfstitelhistorieQueryResponse" : {
-        "type" : "object",
-        "properties" : {
-          "verblijfstitels" : {
-            "type" : "array",
-            "items" : {
-              "$ref" : "#/components/schemas/GbaVerblijfstitel"
+        "allOf" : [ {
```

Comment:
Bij Verblijfstitel was er toch geen sprake van rnideelnemer en verificatie ?

Reply:
Zie nu dat dit is toegevoegd voor categorie 7.

Reply:
@JohanBoer je geeft aan Changes requested. Wat wil je dat wordt aangepast? Of heb je dat ingetrokken?